instructor/ui/instructor.js

```diff
@@ -163,6 +163,30 @@ $(function(){
     $("#id_has_nuclear_fractination").attr('disabled', true).parent().addClass('scb_ab_s_grayed');
     $("#id_has_cytoplasmic_fractination").attr('disabled', true).parent().addClass('scb_ab_s_grayed');
 
+    /**
+     * Western Blotting: band intensity
+     */
+    var sliders = $(".intensity_slider");
+    for(var i=0; i<sliders.length;i++){
+        var $slider=$(sliders[i]);
+        var intensity_dict={0:'', 0.5:'low', 1:'med', 1.5:'high', 2:'extra high'};
+        var intensity=$($slider.siblings('input')[0]).attr('value');
+        $($slider.parent().siblings('.scb_ab_s_intensity_text')).html(intensity_dict[intensity]);
+        $slider.slider({
+          min: 0,
+          max: 2,
+          range: "min",
+          value: intensity,
+          step: 0.5,
+          slide: function( event, ui ) {
+              $(this).siblings()[0].value=ui.value;
+              $($(this).parent().siblings('.scb_ab_s_intensity_text')).html(intensity_dict[ui.value]);
```

Comment:
It looks like this is the same selection as line 174, is there some way to store it in a variable to reuse it?